lib/containers/containerd_crictl.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2020-2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: engine subclass for containerd with crictl specific implementations
+# Maintainer: qac team <qa-c@suse.de>
+
+package containers::containerd_crictl;
+use Mojo::Base 'containers::engine';
+use testapi;
+use containers::common 'install_containerd_when_needed';
+has runtime => 'crictl';
+
+sub init {
+    install_containerd_when_needed();
+
+    my $url = 'https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.23.0/crictl-v1.23.0-linux-amd64.tar.gz';
```

Comment:
Please add a comment that states why we need to download this from an external source. Something like

```suggestion
    # The crictl validation test suite is a plug-in required for this test and needs to be installed from an external source.
    my $url = 'https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.23.0/crictl-v1.23.0-linux-amd64.tar.gz';
```

If I'm completely wrong with what I wrote there, please correct it accordingly.

IMHO every external resource requires a justification comment so that our future-selves don't get confused.